roles/sap_vm_provision/tasks/common/set_ansible_vars.yml

```diff
@@ -19,8 +19,7 @@
     sap_vm_provision_host_specification_plan: "{{ sap_vm_provision_host_specification_plan }}"
     sap_vm_provision_nfs_mount_point: "{{ sap_vm_provision_nfs_mount_point | default('') }}"
     sap_vm_provision_nfs_mount_point_separate_sap_transport_dir: "{{ sap_vm_provision_nfs_mount_point_separate_sap_transport_dir | default('') }}"
-    sap_software_download_directory: "{{ sap_software_download_directory | default('/software') }}"
-    sap_install_media_detect_source_directory: "{{ sap_software_download_directory | default('/software') }}"
+    sap_install_media_detect_source_directory: "{{ sap_install_media_detect_source_directory | default('/software') }}"
```

Comment:
It would be better to keep original` sap_software_download_directory` since it is defined in AP4S vars, and use it as default for `sap_install_media_detect_source_directory`
```yaml
sap_software_download_directory: "{{ sap_software_download_directory | default('/software') }}"
sap_install_media_detect_source_directory: "{{ sap_install_media_detect_source_directory | default(sap_software_download_directory) }}"
```

Reply:
After next release, `sap_software_download_directory` would be removed from AP4S 👍